### DISCUSSION TITLE
[SUGGESTION] Holographic Display

### DISCUSSION THREAD

**Discussion by Servomotor** [tsunamayo/Starship-EVO#4007] [Suggestions]
I know you are busy and have important stuff to complete before you add any more bricks or "fun bits" to the game, but I came across something recently that might have some use in game. 

There is an event going on right now on the official discord where two prisoners have escaped and the community had to make some ships to capture them. 

During the process of building my ship, I wanted to create "scan lines" in the Screens and I had to use a <line-height> adjustment to push the scan lines up. I had accidentally fat fingered and put in 3 digits instead of 2, and it pushed the text off the screen. The text was literally floating above the screen. Very cool effect. 

So I was thinking if we could get some kind of puck brick that had a floating text above it. Basically the same thing as a Screen, but without the "Glass" and instead just a small brick or something used to place it. 

I was thinking there could be a check box or something inside  the command line window that would allow us to have the text "swivel" like name plates work in games. Where you always read the name plate of a player regardless of which way they are facing. this could work as a "rotating" sign of sorts, without any real need for rotation. 

The Holographic Screen might also need some kind of sliders or number inputs to move the text up down, left right, backward or forward. This way the brick itself could be buried if chosen but the screen still legible. 

Anyway, just a thought. 

I know you are super busy, but if you have the time you can see the "Hologram" use of a screen in my build inside the cockpit. 
https://steamcommunity.com/workshop/filedetails/?id=2459532931

You can also see it in use in this video at the 28 second mark. 

https://www.youtube.com/watch?v=Vehk2OYcXro

**Comment by tsunamayo** [comment]
@Servomotor I see, yeah it is a nice idea and something I wanted to do again. But I need a big text screen refactor first, with potentially some link to a letter decal tech and more... Cheers
